www/AndroidDevelopmentEnvironment.md

```diff
@@ -9,7 +9,7 @@ Android Development Environment
 * If you're using android-maven-plugin, `jna.jar` can be used as-is (native libraries will be automatically copied into your project).
 * If you're using Google's Eclipse plugin then you must manually remove libjnidispatch.so from jna.jar/lib/armeabi and add it into your project's libs/armeabi directory.
 * See [http://code.google.com/p/android/issues/detail?id=17861](http://code.google.com/p/android/issues/detail?id=17861) and [http://developer.android.com/guide/practices/jni.html](http://developer.android.com/guide/practices/jni.html) for more information.
-* The NDK can be downloaded from [https://developer.android.com/ndk/index.html]
+* The NDK can be downloaded from <https://developer.android.com/ndk/index.html>
```

Comment:
You can remove the braces - it does not change the rendering (at least on github)